### PR TITLE
Create custom templates for VOA & HGV with custom service YAML parameter to switch between them

### PR DESCRIPTION
**Pull request by kyle-bowden**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -66,6 +67,10 @@ contact.heading=Yr hyn i’w wneud os ydych yn cael anhawster wrth ddefnyddio’
 contact.extra-support.part1='Gallwch '
 contact.extra-support.part2=gysylltu â CThEM os oes angen cymorth ychwanegol
 contact.extra-support.part3=' arnoch i lenwi ffurflenni, neu i gael fformat gwahanol, megis print bras, recordiad sain neu Braille.'
+contact.extra-support-voa.part1='Gallwch '
+contact.extra-support-voa.part2=cysylltwch â VOA am gymorth ychwanegol os oes angen help arnoch i lenwi ffurflenni neu gael fformat gwahanol, fel print bras, recordiad sain neu Braille.
+contact.extra-support-voa.part3=Ffôn: 03000 501 501
+contact.extra-support-voa.part4=Oriau agor: Dydd Llun i Ddydd Gwener: 9:30yb i 2:30yp
```
Comment (kyle-bowden) [inline conf/messages.cy:73]: I'm not sure if this is the correct way to express pm and am in welsh?

File: app/uk/gov/hmrc/accessibilitystatementfrontend/models/AccessibilityStatement.scala [hunk 2]
```diff
@@ -26,6 +26,7 @@ case class AccessibilityStatement(
   serviceDescription: String,
   serviceDomain: String,
   serviceUrl: String,
+  serviceTemplateOverrideType: Option[ServiceTemplateOverrideType] = None,
```
Comment (oscarduignan) [inline app/uk/gov/hmrc/accessibilitystatementfrontend/models/AccessibilityStatement.scala:29]: @kyle-bowden @JoPintoPaul seeing it in use, I wonder if we should change this to "statementTemplate" and make it non-optional

Comment (kyle-bowden) [reply app/uk/gov/hmrc/accessibilitystatementfrontend/models/AccessibilityStatement.scala:29]: @oscarduignan 
I guess the only problem I can see is if it's non-optional, we would then need to add this property to all existing service YAML files.

Comment (oscarduignan) [reply app/uk/gov/hmrc/accessibilitystatementfrontend/models/AccessibilityStatement.scala]: @kyle-bowden could we rename ServiceTemplateOverrideType to TemplateType or StatementTemplateType - it feels like the config is to configure and override, but that override is Some(templateType). Everything else looks good to me 🙌

File: conf/messages.cy (hunk 1 above)
Comment (kyle-bowden) [reply conf/messages.cy:73]: @oscarduignan They never provided a welsh translation, so what I have is a pure google translation. I guess the best thing to do here is to ask them directly?

File: app/uk/gov/hmrc/accessibilitystatementfrontend/models/AccessibilityStatement.scala (hunk 2 above)
Comment (oscarduignan) [reply app/uk/gov/hmrc/accessibilitystatementfrontend/models/AccessibilityStatement.scala]: 🙌

**Pull request closed without merging**